operator/pkg/ciliumidentity/controller.go

```diff
@@ -221,18 +229,23 @@ func (c *Controller) runResourceWorker(context context.Context) error {
 }
 
 func (c *Controller) enqueueReconciliation(item QueuedItem, delay time.Duration) {
+	c.enqueueTimeTracker.Track(item.Key().String())
 	c.resourceQueue.AddAfter(item, delay)
 }
 
 func (c *Controller) processNextItem() bool {
+	processingStartTime := time.Now()
```

Comment:
Shouldn't we move this below at line 244? Currently we are adding the wait for the next element from `resourceQueue` to its processing time.

Reply:
if we move it there, we'll no longer be able to meter the time spent in queue waiting to be processed. 
More exactly the `enqueuedLatency`.